src/dtos/group.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { IsString } from 'class-validator';
+
+export class CreateGroupDto {
+  @IsString()
```

Comment:
Consider adding a @MaxLength or @MinLength decorator from class-validator to ensure that the name field meets certain length requirements, if applicable. It could enhance data validation.